src/frontend/src/components/NavBar/Navbar.jsx

```diff
@@ -29,6 +31,9 @@ export default function NavBar({ username }) {
     }
   }, [isAuthenticated]);
 
+  const isSearchHidden =
```

Comment:
The only thing I see is the height on the background image is dependent on a specific height of the nav bar, and when the search is removed then this is changed and makes the bg image to small.